### PR TITLE
DCOS-45135: Show UI version on the UI settings page

### PR DESCRIPTION
**Pull request by GeorgiSTodorov**
Add a new column showing DC/OS UI Version
to the UI Settings page.

Closes DCOS-45135

## Testing
Go to Settings > UI Settings.
Verify that the 

## Trade-offs
Also added the Chinese translation for "Search bind", since we dropped it somewhere.

## Dependencies
None.

## Screenshots
screenshot from 2019-01-28 12-34-29

### CODE REVIEW

File: src/js/pages/settings/UISettingsPage.js [hunk 1]
```diff
@@ -56,6 +56,16 @@ class UISettingsPage extends React.Component {
...
         <div className="container">
           <ConfigurationMap>
+            <ConfigurationMapHeading className="flush-top">
+              <Trans>DC/OS UI Details</Trans>
+            </ConfigurationMapHeading>
+            <ConfigurationMapSection>
+              <ConfigurationRow
+                keyValue="installedVersion"
+                title={<Trans>Installed Version</Trans>}
+                value={window.DCOS_UI_VERSION}
```
Comment (juliangieseke) [inline src/js/pages/settings/UISettingsPage.js]: 🤷‍♂️

File: locale/zh/messages.json [hunk 2]
```diff
@@ -738,7 +740,7 @@
   "Scheduled": "已计划",
   "Search": "搜索",
   "Search Results": "搜索结果",
-  "Search/bind": "",
+  "Search bind": "搜索 绑定",
```
Comment (juliangieseke) [inline locale/zh/messages.json]: do these relate to this PR? 🤔

Comment (GeorgiSTodorov) [reply locale/zh/messages.json]: Like I said, no, but we lost this translation somewhere, so I added it again.

File: src/js/pages/settings/UISettingsPage.js (hunk 1 above)
Comment (GeorgiSTodorov) [reply src/js/pages/settings/UISettingsPage.js]: Temporary solution.

**Comment by juliangieseke** [comment]
ah, missed that in the PRs description. If thats the case, please make them a separate commit. 🙇

File: locale/zh/messages.json (hunk 2 above)
Comment (juliangieseke) [reply locale/zh/messages.json]: ah, missed that in the PRs description. If thats the case, please make them a separate commit. 🙇

File: src/js/pages/settings/UISettingsPage.js (hunk 1 above)
Comment (nLight) [reply src/js/pages/settings/UISettingsPage.js]: That is what we've decided to go with. The issue wasn't even originally planned so scoping in the data layer would have been a massive creep.

**Pull request merged**